src/engraving/libmscore/scoretree.cpp

```diff
@@ -874,7 +874,7 @@ EngravingObject* BSymbol::scanChild(int idx) const
 
 int BSymbol::scanChildCount() const
 {
-    return _leafs.size();
+    return static_cast<int>(_leafs.size());
```

Comment:
I believe, better change method signature to `size_t BSymbol::scan ChildCount() const`

Reply:
I general I tend to agree, but that requires more changes in lots other places too

Reply:
See b52db31

Reply:
You wanna go that way?

Reply:
I believe it's worth it

Reply:
This one indeed might. Also it is done ;-)
Not so the other one, below.